php_phongo.c

```diff
@@ -1886,6 +1909,25 @@ PHP_MINIT_FUNCTION(mongodb)
 	phongo_std_object_handlers.get_closure          = NULL;
 	*/
 
+	/* Initialize zend_class_entry dependencies.
+	 *
+	 * Although DateTimeImmutable was introduced in PHP 5.5.0,
+	 * php_date_get_immutable_ce() is not available in PHP versions before
+	 * 5.5.24 and 5.6.8.
+	 *
+	 * Although JsonSerializable was introduced in PHP 5.4.0,
+	 * php_json_serializable_ce is not exported in PHP versions before 5.4.26
+	 * and 5.5.10. For later PHP versions, looking up the class manually also
+	 * helps with distros that disable LTDL_LAZY for dlopen() (e.g. Fedora).
+	 */
+	php_phongo_date_immutable_ce = php_phongo_fetch_internal_class(ZEND_STRL("datetimeimmutable") TSRMLS_CC);
+	php_phongo_json_serializable_ce = php_phongo_fetch_internal_class(ZEND_STRL("jsonserializable") TSRMLS_CC);
+
+	if (php_phongo_json_serializable_ce == NULL) {
+		zend_error(E_ERROR, "JsonSerializable class not defined. Please ensure that the json extension is loaded before the mongodb extension.");
```

Comment:
Assuming our MINIT is only ever called through [`zend_startup_module_ex()`](https://github.com/php/php-src/blob/PHP-7.0.0/Zend/zend_API.c#L1794), I agree that this is probably redundant. The class is always available in PHP 5.4.0+ (https://github.com/php/php-src/commit/1381b14eaaa6d0f3dcbc9b9a2c8116c2ecfac2c4), so we should never fail to find it on any supported PHP version. Should I remove the check entirely?